src/platform/qt/ts/mgba-zh_CN.ts

```diff
@@ -11,17 +11,17 @@
     <message>
         <location filename="../AboutScreen.ui" line="23"/>
         <source>&lt;a href=&quot;http://mgba.io/&quot;&gt;Website&lt;/a&gt; • &lt;a href=&quot;https://forums.mgba.io/&quot;&gt;Forums / Support&lt;/a&gt; • &lt;a href=&quot;https://patreon.com/mgba&quot;&gt;Donate&lt;/a&gt; • &lt;a href=&quot;https://github.com/mgba-emu/mgba/tree/{gitBranch}&quot;&gt;Source&lt;/a&gt;</source>
-        <translation>&lt;a href=&quot;http://mgba.io/&quot;&gt;主页&lt;/a&gt; • &lt;a href=&quot;https://forums.mgba.io/&quot;&gt;论坛 / 支持&lt;/a&gt; • &lt;a href=&quot;https://patreon.com/mgba&quot;&gt;捐助&lt;/a&gt; • &lt;a href=&quot;https://github.com/mgba-emu/mgba/tree/{gitBranch}&quot;&gt;源代码&lt;/a&gt;</translation>
+        <translation>&lt;a href=&quot;http://mgba.io/&quot;&gt;网站&lt;/a&gt; • &lt;a href=&quot;https://forums.mgba.io/&quot;&gt;论坛、支持&lt;/a&gt; • &lt;a href=&quot;https://patreon.com/mgba&quot;&gt;捐助&lt;/a&gt; • &lt;a href=&quot;https://github.com/mgba-emu/mgba/tree/{gitBranch}&quot;&gt;源代码&lt;/a&gt;</translation>
     </message>
     <message>
         <location filename="../AboutScreen.ui" line="41"/>
         <source>Branch: &lt;tt&gt;{gitBranch}&lt;/tt&gt;&lt;br/&gt;Revision: &lt;tt&gt;{gitCommit}&lt;/tt&gt;</source>
-        <translation>分支: &lt;tt&gt;{gitBranch}&lt;/tt&gt;&lt;br/&gt;修订: &lt;tt&gt;{gitCommit}&lt;/tt&gt;</translation>
+        <translation>分支: &lt;tt&gt;{gitBranch}&lt;/tt&gt;&lt;br/&gt;修订版: &lt;tt&gt;{gitCommit}&lt;/tt&gt;</translation>
     </message>
     <message>
         <location filename="../AboutScreen.ui" line="58"/>
         <source>{projectName}</source>
-        <translation>{projectName}</translation>
```

Comment:
Please don't remove strings like this. That applies to all of the strings that were arbitrarily blanked out in this PR (of which there were a lot).